ntc_templates/templates/cisco_ios_show_cts_pacs.textfsm

```diff
@@ -0,0 +1,34 @@
+Value AID (\S+)
+Value PAC_TYPE (.*)
+Value INITIATOR_ID (\S+)
+Value AUTHENTICATOR_ID_INFO (.*)
+Value CREDENTIAL_LIFETIME (.*)
```

Comment:
*Regarding credential lifetime:*
Capture the time, next pattern of three (tz, day of week, month), then numeric day, and optionally the year if it appears. All while excluding the occasional trailing closing parenthesis `)` or extra odd characters (ex: `!W[^Ke`).

If you're open to this suggestion, click commit suggestion, and also regenerate the structured yaml to match (since it will exclude those chars). Thank you!

```suggestion
Value CREDENTIAL_LIFETIME (\S+\s+(?:\w+\s*?){3}\d+(?:\s*\d+)?)
```